ios/traversal/ViewHierarchyNavigator.swift

```diff
@@ -91,6 +91,8 @@ public class ViewHierarchyNavigator: NSObject {
       return validTextInput
     }
 
+    guard String(describing: type(of: view)) != "KeyboardToolbarExcludeView" else { return nil }
```

Comment:
1. is it possible to use instance check instead of comparing two strings?
2. it repeats twice